### DISCUSSION TITLE
No PATs given to me after generation

### DISCUSSION THREAD

**Discussion by BANOnotIT** [community/community#36344] [Other Feature Feedback, Questions, & Ideas]
I've generated new token with needed params
![image](https://user-images.githubusercontent.com/11363215/196057489-5ebe2db5-f472-4c14-8b6a-4677f9c91cc1.png)

Click on "generate token"
![image](https://user-images.githubusercontent.com/11363215/196057497-a3973f45-5413-4892-8259-1d0903fde3b9.png)

And instantly redirected to PATs page without any token given to me.

![image](https://user-images.githubusercontent.com/11363215/196057529-2229915b-9c29-4854-b895-d661b4e7670e.png)

What should I do to get my token?

**Comment by ossanna16** [comment]
Hey there @BANOnotIT and thank you for asking a good question! I'm glad you found our community 🙂
